kolibri/core/assets/src/mixins/commonCoreStrings.js

```diff
@@ -1023,6 +1023,90 @@ export const coreStrings = createTranslator('CommonCoreStrings', {
     message: 'Folders',
     context: "Tab label in the 'Browse channel' page that allows to navigate through its topics.",
```

Comment:
This sounds good. @radinamatic this is actually not from the past papers work - is this something we want to change for 0.16? If so, I will add it to my upcoming PR.

Reply:
I'm going to leave this out until we get to a more fully functional 0.16, instead of this one past papers feature, and I'll add this as an issue that's a todo for myself when we get to this point.